content/people/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: People
+---
+
+At FOSDEM 2025, a few people got together who have committed themselves to driving the 
+fediverse forward. This group of fedi's continues to grow. Here we try to maintain an 
+up-to-date list (sorted alphabetically).
+
+| Name     					| Fedi address						| Website											|
+| ------------------------- | --------------------------------- | ------------------------------------------------- |
+| Andy Piper				| @andypiper@macaw.social			| https://andypiper.me								|
+| Casey Kolderup			| @casey@sharetron.com				| https://casey.kolderup.org						|
+| Charles Iliya Krempeaux	| @reiver@mastodon.social			| https://reiver.link								|
+| Fredy       				| @naturzukunft@mastodon.social		| https://rdf-pub.org								|
+| Julian Lam				| @julian@community.nodebb.org		| https://github.com/NodeBB/NodeBB					|
+| Liaizon Wakest			| @liaizon@wake.st					| https://wake.st									|
+| Matt Baer					| @matt@writing.exchange			| https://writefreely.org							|
+| Matthias Pfefferle		| @pfefferle@mastodon.social		| https://notiz.blog								|
+| Sofía Aritz				| @sofiaritz@hachyderm.io			| https://sofiaritz.com								|
+| Tommi						| @tommi@pan.rent					| https://tommi.space								|
+| ?							| @funkwhale@social.funkwhale.audio	| https://dev.funkwhale.audio/funkwhale/funkwhale	|
```

Comment:
Because this is about "People" I would recommend removing this until we have a name!?!

```suggestion
```